alkira/resource_alkira_segment.go

```diff
@@ -116,12 +116,34 @@ func resourceSegmentDelete(d *schema.ResourceData, m interface{}) error {
 }
 
 func generateSegmentRequest(d *schema.ResourceData) (*alkira.Segment, error) {
+
 	seg := &alkira.Segment{
-		Asn:     d.Get("asn").(int),
-		IpBlock: d.Get("cidr").(string),
-		Name:    d.Get("name").(string),
+		Asn:  d.Get("asn").(int),
+		Name: d.Get("name").(string),
 		ReservePublicIPsForUserAndSiteConnectivity: d.Get("reserve_public_ips").(bool),
 	}
 
+	// User may include 1 or more cidrs. Set them accordingly
+	cidrs := convertTypeListToStringList(d.Get("cidrs").([]interface{}))
+
+	if len(cidrs) == 1 {
```

Comment:
Why do we need this? I thought `IpBlock` was deprecated... We only need to care `IpBlocks`... nope?